src/app/common/money/calculate-money.spec.ts

```diff
@@ -1,7 +1,7 @@
+import { createMoney, createMoneyFromDecimal } from '@leather-wallet/utils';
```

Comment:
@kyranjamie wasn't there discussion about moving `createMoney` and `createMoneyFromDecimal` to the models pkg so it lives with the `Money` model?

Reply:
hm, I think like in `models` should live only `models` no?

Reply:
We had discussed (I think) having the functions that create the models live with the models, see `createMarketData` in `market.model.ts`.

Reply:
I see, makes sense

Reply:
I think it's okay for "model creation" fns to live with the models, but not logic more complicated than this